config/config.php

```diff
@@ -11,6 +11,7 @@
     | ( For now only used in the alternate-hreflang cmponent )
     |
     */
-    'prefixDefaultLocale' => true
+    'prefixDefaultLocale' => true,
+    'cacheTimeout'        => 1440,
```

Comment:
You need to add a descriptive comment detailing what this setting is, why someone might want to change it, and what the possible values are before I can merge this.